eosmetrics/emtr-event-recorder.c

```diff
@@ -319,12 +339,22 @@ send_events_to_dbus (EmtrEventRecorder *self,
     emtr_event_recorder_get_instance_private (self);
 
   GVariantBuilder uuid_builder;
-  get_uuid_builder (parsed_event_id, &uuid_builder);
-  GVariant *event_id_variant = g_variant_builder_end (&uuid_builder);
+  GVariant *event_id_variant;
```

Comment:
To avoid allocating the `GVariantBuilder` (and friends) and then immediately freeing them in the `disable_event_submission () == TRUE` case.